app/models/vm_or_template.rb

```diff
@@ -918,7 +918,14 @@ def log_all_proxies(all_proxy_list, message)
   end
 
   def log_proxies_vm_config
-    "[#{log_proxies_format_instance(self)}] on host [#{log_proxies_format_instance(host)}] #{ui_lookup(:table => "storages").downcase} [#{storage.name}-#{storage.store_type}]"
+    msg = "[#{log_proxies_format_instance(self)}] on host [#{log_proxies_format_instance(host)}] "\
+    "#{ui_lookup(:table => "storages").downcase} "
+    msg += if storage
+             "[#{storage.name}-#{storage.store_type}]"
+           else
+             "No storage"
+           end
+    msg
```

Comment:
@Fryguy OK, it does look cleaner, will change it in PR removing not used `log_proxies`